Makefile.am

```diff
@@ -164,15 +164,14 @@ test_network_test_LDADD = librauctest.la
 test_progress_test_SOURCES = test/progress.c
 test_progress_test_LDADD = librauctest.la
 
-edit = sed \
+SED_REPLACE = $(SED) \
        -e 's|@bindir[@]|$(bindir)|g' \
        -e 's|@datadir[@]|$(pkgdatadir)|g' \
        -e 's|@prefix[@]|$(prefix)|g'
 
 data/%: data/%.in
-	rm -f $@ $@.tmp
-	$(edit) '$(srcdir)/$@.in' >$@.tmp
-	mv $@.tmp $@
+	$(MKDIR_P) $(dir $@)
```

Comment:
Use $(AM_V_GEN) here and add AC_PROG_MKDIR_P, AC_PROG_SED to configure.ac. See systemd for reference.